ft-sys/src/diesel_sqlite/sqlite_value.rs

```diff
@@ -37,6 +37,13 @@ impl<'a> SqliteValue<'a> {
             _ => Err("Unexpected type".into()),
         }
     }
+
+    pub(crate) fn jsonb(&self) -> diesel::deserialize::Result<serde_json::Value> {
+        match self.raw_value {
+            Value::Jsonb(i) => Ok(serde_json::from_slice(i)?),
+            _ => Err("Unexpected type".into()),
```

Comment:
Update these error messages to include the "got type" part also, so it's easier to debug.